spm-monitor-starter/src/main/java/com/sematext/spm/client/MonitorAgentBootstrapThread.java

```diff
@@ -113,6 +113,12 @@ public void run() {
 
       System.out.println("Monitor config file is : " + monitorConfigFile);
       Properties monitorProps = MonitorUtil.loadMonitorProperties(monitorConfigFile);
+
+      if(monitorProps == null) {
+        System.out.println("Unable to open monitor config file.");
```

Comment:
Include the path so one can tell which config file we tried to open

Reply:
+1